packages/governance/src/types.js

```diff
@@ -1,77 +1,128 @@
 // @ts-check
 
 /**
- * @typedef { 'amount' | 'brand' | 'installation' | 'instance' | 'nat' | 'ratio' | 'string' | 'unknown' } ParamType
+ * @typedef { 'choose_n' | 'order' | 'weight' } ChoiceMethod
+ *  CHOOSE_N: This is "approval voting". The voter specifies some number of
+ *    positions, and is endorsing them equally.
+ *  ORDER: The voter assigns ordinal numbers to some of the positions. The
+ *    positions will be treated as an ordered list with no gaps.
+ *  WEIGHT: The voter assigns a distinct integer weight to some number of
+ *   positions. The weights are not required to be distinct or consecutive.
+ *   High numbers are most preferred.
```

Comment:
Formatted this way, vscode hover renders this as a markdown bulleted list. Otherwise it just ran all the text together into a big paragraph.

```suggestion
 *  * CHOOSE_N: This is "approval voting". The voter specifies some number of
 *    positions, and is endorsing them equally.
 *  * ORDER: The voter assigns ordinal numbers to some of the positions. The
 *    positions will be treated as an ordered list with no gaps.
 *  * WEIGHT: The voter assigns a distinct integer weight to some number of
 *    positions. The weights are not required to be distinct or consecutive.
 *    High numbers are most preferred.
```

Reply:
thanks. Done.